shopfloor_packing_info/migrations/14.0.1.2.0/post-migrate.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import logging
+
+from odoo import SUPERUSER_ID, api
+
+_logger = logging.getLogger(__name__)
+
+# NOTE: This module is deprecated in favor of stock_picking_partner_note.
```

Comment:
We should move this upgrade into shopfloor and ignore this module. Just keep the msg that is deprecated.
As we migrate the data we can flag it as to be uninstalled.

Reply:
@jbaudoux what do you think?

Reply:
Nevermind about this. However, IMO we should either uninstall the module automatically or at least add a post load hook to log deprecation.